### PR TITLE
Add intro text for Computations section

### PR DESCRIPTION
**Pull request by paigem**
This PR does three things:
- Add text for intro to computations section
- Update name of section: Large-scale data analysis --> Data analysis of large-scale datasets
- Update wording in computations notebook

Questions before merging: It appears that there are separate files on chunking and analysis on the time dimension in the [/computations](https://github.com/ACDguide/BigData/tree/main/BigData/computations) folder, but they have not been added to the book's table of contents. See Issue #89

### CODE REVIEW

File: BigData/computations/computations-intro.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Large-scale data analysis
 
-Overview
+## Overview
+This section includes examples of common types of computations performed on large, climate datasets. We focus on the theory of how these computations are handled on chunked datasets when using tools like `Xarray` and `Dask`. We then provide examples of the specific functions that are often used to carry out each computation and, where possible, demonstrations of these tools in use.
+
+## How is analysis on large-scale data different from that on smaller datasets?
+Large datasets are often too big to load into RAM on the computer or server that you use to do your analysis. However, we often don't need to do a computation on the entire dataset. The previous section on large-scale climated data discussed how large datasets are typically saved in "chunks". Analysis on large-scale datasets also makes use of these chunks in order to apply computations only on the portion of the dataset needed for that particular calculation.
```
Comment (hot007) [inline BigData/computations/computations-intro.md:7]: verify that the "previous section" really does cover chunking following restructure :)

File: BigData/_toc.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ parts:
     - file: data/data-format.md
     - file: data/data-netcdf.md
     - file: data/data-zarr.md
-  - caption: Large-scale data analysis
+  - caption: Analysis of large-scale datasets
```
Comment (hot007) [inline BigData/_toc.yml:11]: Looks good to me

File: BigData/computations/computations-intro.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Large-scale data analysis
 
-Overview
+## Overview
+This section includes examples of common types of computations performed on large, climate datasets. We focus on the theory of how these computations are handled on chunked datasets when using tools like `Xarray` and `Dask`. We then provide examples of the specific functions that are often used to carry out each computation and, where possible, demonstrations of these tools in use.
```
Comment (hot007) [inline BigData/computations/computations-intro.md:4]:
```suggestion
This section includes examples of common types of computations performed on large climate datasets. We focus on the theory of how these computations are handled on chunked datasets when using tools like `xarray` and `dask`. We then provide examples of the specific functions that are often used to carry out each computation and, where possible, demonstrations of these tools in use.
```

Comment (hot007) [reply BigData/computations/computations-intro.md:4]: I think libraries are lower case but ignore that if not! Also I removed a comma that I felt broke it up too much.

**Review by hot007 — APPROVED** [review]
This all looks good to me, thanks @paigem !

File: BigData/computations/computations-intro.md (hunk 1 above)
Comment (paigem) [inline BigData/computations/computations-intro.md:7]:
```suggestion
Large datasets are often too big to load into RAM on the computer or server that you use to do your analysis. However, we often don't need to do a computation on the entire dataset. The [section on large-scale climate data](https://acdguide.github.io/BigData/data/data-netcdf.html) discusses how large datasets are typically saved in "chunks". Analysis on large-scale datasets also makes use of these chunks in order to apply computations only on the portion of the dataset needed for that particular calculation.
```

Comment (paigem) [reply BigData/computations/computations-intro.md:7]: Very good point! I updated to just remove that language so we aren't stuck with any specific section ordering. I also added a link to that section.

**Pull request merged**